x-pack/plugins/apm/server/routes/alerts/register_transaction_duration_alert_type.ts

```diff
@@ -188,6 +192,24 @@ export function registerTransactionDurationAlertType({
           windowSize: ruleParams.windowSize,
           windowUnit: ruleParams.windowUnit,
         });
+
+        // The function args wrapped with arrays because this function is shared with the frontend and the vars there are parsed by parseTechnicalFields()
```

Comment:
Might be good to change the helper so it doesn't accept an array

Reply:
As mentioned here https://github.com/elastic/kibana/pull/128243#discussion_r833059381. The arrays are the output of `parseTechnicalFields` so I believe we have to.

Reply:
We could just pass the first element from the ui side instead, but not too precious about it

Reply:
Related to https://github.com/elastic/kibana/pull/128243#discussion_r833159321

Reply:
Thanks!